IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the money stuck rate of ticket.
```

Comment:
```suggestion
Indicate the frequency of money being stuck or jammed in appliance.
```

Reply:
Updated, but changed "frequency" to "ratio". https://github.com/bSI-InfraRoom/IFC-Specification/pull/160/commits/8b12068ba20d88a43f933b941b15b54fce5e5c4d